src/rebar3_ex_doc.erl

```diff
@@ -486,3 +492,15 @@ help(language) ->
 help(logo) ->
     "Path to the image logo of the project (only PNG or JPEG accepted)."
     "The image size will be 64x64 and copied to the assets directory.".
+
+% Mermaid support
+mermaid_add(false) ->
+  "";
+mermaid_add(true) ->
+  mermaid_add(?DEFAULT_MERMAID_VSN);
+mermaid_add(Vsn) when is_list(Vsn) ->
+    [{before_closing_head_tag, #{
+        html => "<script src=\"https://cdn.jsdelivr.net/npm/mermaid@" ++ Vsn ++ "/dist/mermaid.min.js\" />"
```

Comment:
I wonder if we should refer to the latest? I also wonder, if we should let the use provide an option to specify a ver? Maybe overkill?

Reply:
There is such an option; it's this implementation 😄

Reply:
> I wonder if we should refer to the latest?

That was my goal: I didn't check what "latest" was available (or if a `latest` tag is available either). In any case, we could do that, I'm just cautious with version stuff usually, because I don't want it to break easily. I can propose a change here... a minute.

Reply:
6482004